templates/song_rater/add_song.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Add_Song</title>
+</head>
+<body>
+<h1>Add a song</h1>
+<form action="" method="POST">
+   {% csrf_token %}
+   Title: <input type="text" name="title"/>
+   <br/>
+   Artist: <input type="text" name="artist"/>
+   <br/>
+   <br/>
+   <input type="submit" value="Post"/>
+    <li><a href="{% url 'song_rater:index'%}">Back to homepage</a></li>
```

Comment:
Nice navigation link.

I wouldn't put in inside a `li` tough.  I think it looks weird with only one option.  Up to style tough.

Reply:
Thanks